pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerImpl.java

```diff
@@ -220,13 +220,13 @@ protected ConsumerImpl(PulsarClientImpl client, String topic, ConsumerConfigurat
         }
 
         this.connectionHandler = new ConnectionHandler(this,
-        		        new BackoffBuilder()
-        	               .setInitialTime(100, TimeUnit.MILLISECONDS)
-        	               .setMandatoryStop(60, TimeUnit.SECONDS)
-        	               .setMax(0, TimeUnit.MILLISECONDS)
-        	               .useUserConfiguredIntervals(backoffIntervalNanos, 
-        	                                           maxBackoffIntervalNanos)
-        	               .create(),
+                      new BackoffBuilder()
```

Comment:
try to avoid formatting the code as reformatting will make reviewers harder to review your code.